automation_tools/__init__.py

```diff
@@ -1026,6 +947,129 @@ def product_install(distribution, create_vm=False, certificate_url=None,
         )
 
 
+def satellite5_product_install(create_vm=False, selinux_mode=None):
+    """Task which installs only satellite5 product.
+
+    If ``create_vm`` is True then ``vm_destroy`` and ``vm_create`` tasks will
+    be run. Make sure to set the required environment variables for those
+    tasks. Also, if one of the ``setup_ddns`` required environment variables
+    is set then that task will run.
+
+    :param bool create_vm: creates a virtual machine and then install the
+        product on it. Default: False.
```

Comment:
Documentation generators such as Sphinx typically are smart enough to examine the arguments to a method and show the default in the generated docstring. For example, see [this method](http://nailgun.readthedocs.org/en/latest/api/nailgun.html#nailgun.config.BaseServerConfig.get).